modernizer-annotation-processor/src/main/java/org/gaul/annotation_processsor/ModernizerAnnotationProcessor.java

```diff
@@ -53,36 +57,56 @@ public final boolean process(
         RoundEnvironment roundEnv
     ) {
         for (TypeElement annotation : annotations) {
-            List<String> annotatedClasses =
-                getAnnotatedClasses(roundEnv, annotation);
-            makeAnnotatedClassesFile(annotatedClasses);
+            List<String> annotatedClasses = new ArrayList<String>();
+            List<String> annotatedMethods = new ArrayList<String>();
+            getAnnotatedElements(
+                roundEnv, annotation, annotatedClasses, annotatedMethods);
+            makeAnnotatedElementsFiles(annotatedClasses, annotatedMethods);
         }
         return true;
     }
 
-    private List<String> getAnnotatedClasses(
+    private void getAnnotatedElements(
         RoundEnvironment roundEnv,
-        TypeElement annotation
+        TypeElement annotation,
+        List<String> annotatedClasses,
+        List<String> annotatedMethods
```

Comment:
Rather than passing a list and having the method modify that list, can we instead make the method return an object with info about all the various suppressions?